src/gui/PatternsManagerWindow.cpp

```diff
@@ -74,12 +85,11 @@ void PatternsManagerWindow::create_pattern()
 
     const auto base_pfs = to_u8string(base_pfs_qs);
 
-    auto all_pfs            = settings_.current_profile().per_file_settings();
     PerFileSettings new_pfs = **std::ranges::find(all_pfs, base_pfs, [](const PerFileSettings *pfs) {
         return pfs->pattern.text();
     });
 
-    new_pfs.pattern = Pattern{base_pfs};
+    new_pfs.pattern = Pattern{new_pfs_name, Pattern::Type::Regex};
```

Comment:
Why regex? I tend to default to wildcard, and regex could be an option

Reply:
I will be honest, I don’t really understand how wildcard patterns are supposed to work. 

Letting user choose would definitely be best.

Reply:
Wildcards are actually really simple compared to regex. Especially bethutil's implementation, which is very limited
- `?` matches a single character. Equivalent to regex `.`
- `*` matches unlimited characters. Equivalent to regex `.*?`
- `[ab]` matches `a` or `b`. Same as in regex

That's all